src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml

```diff
@@ -23,111 +23,114 @@ config:
     endpoint: 0
 
 tests:
-    - label: "Commission TH and DUT over BLE to setup the Wi-Fi"
-      verification: |
-
-      disabled: true
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
 
     - label: "TH reads the ClusterRevision from DUT"
-      verification: |
-          ./chip-tool networkcommissioning read cluster-revision  1 0
-
-          Verify on the TH Log:
-
-          [1649244866.275684][30440:30446] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0031 Attribute 0x0000_FFFD DataVersion: 2702518944
-          [1649244866.277721][30440:30446] CHIP:TOO:   ClusterRevision: 1
-          [1649244866.277827][30440:30446] CHIP:EM: Sending Standalone Ack for MessageCounter:10092551 on exchange 64234i
-      disabled: true
+      cluster: "Network Commissioning"
```

Comment:
Could just set that as the default value in the test config section above, right?